binr/radiff2/radiff2.c

```diff
@@ -166,6 +166,7 @@ static int show_help(int v) {
 		"  -r         output in radare commands\n"
 		"  -s         compute text distance\n"
 		"  -ss        compute text distance (using levenstein algorithm)\n"
+		"  -S [column_name]	Sort by <column_name>\n"
```

Comment:
just use [name] and the description should follow the same syntax as the others

* aligned to the same column
* enumerate all the possible column names
* starting in lowercase